StyleGuide.md

```diff
@@ -1070,6 +1120,26 @@ Use the following HTML and Liquid to include an image in a Markdown page:
 
 Example: [Decommission Nodes](https://www.cockroachlabs.com/docs/stable/remove-nodes.html#step-1-check-the-node-before-decommissioning)
 
+<a name="videos"></a>
+
+### Videos
+
+Like images, use videos to clarify a topic, but only use them as needed. Typically, videos should be hosted on the official [CockroachDB YouTube page](https://www.youtube.com/@cockroachdb).
+
+Use the following Liquid to include an embedded video in a Markdown page:
+
+~~~ md
+{% include_cached youtube.html video_id="<YouTube ID>" [widescreen=true] %}
+~~~
+
+The `video_id` parameter is required. The `widescreen` parameter is optional and meant for videos with wide banners such as our video on [Foreign Key Constraints](https://www.youtube.com/watch?v=5kiMg7GXAsY).
```

Comment:
Maybe clarify that the video_id is the value after v= in the URL?

Reply:
Done!